mec/pkg/model/types.go

```diff
@@ -29,19 +29,21 @@ type ImageRef struct {
 }
 
 type Manifest struct {
-	SchemaVersion ManifestSchemaVersion `json:"schemaVersion"`
-	Name          string                `json:"name"`
-	Description   string                `json:"description"`
-	Version       string                `json:"version"`
-	Phase         v1alpha1.FilterPhase  `json:"phase"`
-	Priority      int                   `json:"priority"`
-	Module        string                `json:"module"`
+	SchemaVersion ManifestSchemaVersion `yaml:"schemaVersion"`
```

Comment:
not sure if it matters, but I think all of the common yaml libraries use the `json` and `yaml` tags equivalently.

Reply:
not quite, apparently. the unmarshalling of `ManifestSchemaVersion` didn't work previously

Reply:
interesting. Thanks.